types/webassemblyjs__ast/index.d.ts

```diff
@@ -0,0 +1,129 @@
+// Type definitions for webassemblyjs__ast x.x
+// Project: https://github.com/baz/foo (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: Mohsen Azimi <https://github.com/mohsen1>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "@webassemblyjs/ast" {
```

Comment:
Just remove this `declare module` wrapper, the file itself declares the module.

Reply:
This is special because module name is `"@webassemblyjs/ast"`

Reply:
TypeScript should check for `foo__bar` when it sees an import from `@foo/bar`, so it should work without the `declare module`.